client/src/header/components/Header.tsx

```diff
@@ -45,7 +45,10 @@ const Header: FC<Props> = ({
           <Link to='/'>Home</Link>
         )}
       </Menu.Item>
+
       {isLoggedIn && <Menu.Item onClick={onLogout}>Sign Out</Menu.Item>}
+
+      {window.location.hostname == 'localhost' && <Menu.Item><Link to='/testing'>Testing</Link></Menu.Item>}
```

Comment:
This should work, but of course there's no way to test it until it's deployed :-)